### PR TITLE
GitHub Actions

### PR DESCRIPTION
**Pull request by Dahlgren**
**When merged this pull request will:**
- Add HEMTT project specification
- Build Arma 3 addons using HEMTT (no binarization of p3ds)
- Build Arma 3 extension, 32 bit + 64 bit
- Build Teamspeak plugin, 32 bit + 64 bit
- Upload archive with everything packed together

### CODE REVIEW

**Comment by Dahlgren** [comment]
Example build, https://github.com/Dahlgren/task-force-arma-3-radio/runs/1400352630

**Comment by jokoho48** [comment]
why not use the offical hemtt github action? https://github.com/arma-actions/hemtt

**Comment by Dahlgren** [comment]
It (currently) cannot be locked to specific HEMTT version and new releases are not always backwards compatible. For example the 0.7 versions do not support json project files anymore. Using a specific version avoids suddenly breaking the TFAR builds.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+      - name: Git checkout
+        uses: actions/checkout@v2
+        with:
+          lfs: true
+
+      - name: HEMTT Build
+        uses: docker://hemtt/hemtt:0.7.6
+        with:
+          entrypoint: hemtt
+          args: build --release
```
Comment (Dahlgren) [inline .github/workflows/ci.yml:35]: `--ci` should be added here for less fancy terminal output

**Comment by dedmen** [comment]
No need, appveyor does it all correctly.
And this is missing stuff like dll signing, correct build versions, pdb upload for dll's (which can be fixed)
Also how does hemtt binarize p3d's without Arma tools being available?
Also you cannot rebuild the Arma extensions. BattlEye.

**Pull request closed without merging**